CodingChallenge.PirateSpeak/Solution.cs

```diff
@@ -1,13 +1,44 @@
 ﻿using System;
 using System.Linq;
+using System.Collections.Generic;
 
 namespace CodingChallenge.PirateSpeak
 {
     public class Solution
     {
         public string[] GetPossibleWords(string jumble, string[] dictionary)
         {
-            throw new NotImplementedException();
+            /* Order jumble by characters in asec, this returned an IOrderedEnumerable. String.Join joins the IOrderedEnumerable
+               into a string. The ToLower() is to account for any uppercase characters that may be passed. */
+
+            string sortedJumble = (String.Join("", jumble.OrderBy(x => x))).ToLower();
+
+            /* Looks like Array.Append() as been removed? So need to convert the string array to a List in order to add items to it
+               possibleWords will contain a list of potential mumblings the pirate could have been saying */
+
+            List<string> possibleWords = new List<string>();
+
+            // Loop over each word in the supplied dictionary
+
+            foreach (string word in dictionary)
```

Comment:
This is a great solution, just out of curiosity, can you think of a way to do this using linq instead of a loop?

Reply:
Will work on completing this challenge with Linq now